op-acceptance-tests/tests/base/rpc_connectivity_test.go

```diff
@@ -3,66 +3,73 @@ package base
 import (
 	"context"
 	"fmt"
+	"math/big"
 	"testing"
 	"time"
 
-	"github.com/ethereum-optimism/optimism/devnet-sdk/system"
-	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
 	"github.com/ethereum-optimism/optimism/op-service/testlog"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/stretchr/testify/require"
 )
 
 // TestRPCConnectivity checks we can connect to L2 execution layer RPC endpoints
-func TestRPCConnectivity(t *testing.T) {
-	systest.SystemTest(t,
-		rpcConnectivityTestScenario(),
-	)
-}
+func TestRPCConnectivity(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewMinimal(t)
+	logger := testlog.Logger(t, log.LevelInfo).With("Test", "TestRPCConnectivity")
```

Comment:
Not blocking the PR on this, but nit: you can do `logger := t.Logger()`, and that will then be annotated with the test name etc. for you.

Reply:
Good to know, thanks!